### DISCUSSION TITLE
Webhook is not working as expected after setting it up via command line

### DISCUSSION THREAD

**Discussion by prashantcct** [def-studio/telegraph#583] [Q&A]
We have integrated bot in laravel using this package.

And after sometime, we have revoke the token and generate new token for the same bot, using that same token we have created new bot and updated web hook for the same and we confirm that URL which has been given for web hook is working fine.

But by doing so we are getting this error after setting up web hook and in bot its showing unknown command.
![MicrosoftTeams-image](https://github.com/defstudio/telegraph/assets/20067505/dab3c17f-8eca-45f8-8014-c88e1196cabb)

Anyone please let us know, if same has issue occured? If we are missing something and or needs to change any thing?

Already tried things as below
- We upgraded package to latest version
- We have remove existing bot from db and created new one using command line, commands are as below
setup new bot :
php artisan telegraph:new-bot
setup web hook
php artisan telegraph:set-webhook

**Comment by fabio-ivona** [comment]
@prashantcct thanks for using our package :heart: 

> [!WARNING]
> you should keep your token secret, so I advise you to generate a new one as it has been disclosed in your previous message.

it seems that Telegram can't find your webhook (your server answered `404 not found`)

I think this could not find a bot with that token:

```php
//DefStudio\Telegraph\Models\TelegraphBot::89
 return self::query()->where('token', $token)->sole();
```

could you put a \Log::debug($token) to check the value of `$token` you receive, and if it matches with the one in the URL specified in your image (or the new one you generate)?
